olive/passes/onnx/quantization.py

```diff
@@ -610,13 +642,58 @@ def _run_for_config(
 
         output_model_path = resolve_onnx_path(output_model_path, Path(model.model_path).name)
 
-        quant = MatMul4BitsQuantizer(
-            model.load_model(), config["block_size"], config["is_symmetric"], config["nodes_to_exclude"]
-        )
+        if version.parse(OrtVersion) >= version.parse("1.17.0"):
+            from onnxruntime.quantization.matmul_4bits_quantizer import WeightOnlyQuantConfig
+
+            if config["algorithm"] in ("RTN", "GPTQ"):
```

Comment:
I suppose the underlying ORT will do raise the ImportError and we might need not this check?

Reply:
Yes, the import error will be raised in ort as well. But no extra info to tell which config needs inc package. 
Maybe we can keep it here to provide clear description once user choose the wrong config.